Copy update: refresh managed infrastructure

jw-can opened the pull request:
## Done

- moved `managed/index.html` to `managed-infrastructure/` and copied related files for "contact us" form
- updated copy of main page
- removed obsolete redirect `managed/cassandra`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- visit http://localhost:8001/managed-infrastructure
- verify changes according to [copy doc](https://docs.google.com/document/d/19GoMudyrd9Zt1e7kfj-koL-Lt0RGYzWHRPuuKYUPvqw/edit?tab=t.0)

## Issue / Card

https://warthogs.atlassian.net/browse/WD-30628?focusedCommentId=947671&sourceType=mention


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

webteam-app commented:
### [<img src='https://assets.ubuntu.com/v1/6baef514-ubuntu-circle-of-friends-large.svg' height=32 width=32> Demo</img>](https://ubuntu-com-15807.demos.haus)
### [<img src='https://assets.ubuntu.com/v1/e512b0e2-jenkins.svg' height=32 width=32> Jenkins </img>](https://jenkins.canonical.com/webteam/job/start-demo/31777)
### [<img src='https://assets.ubuntu.com/v1/7144ec6d-logo-jaas-icon.svg' height=32 width=32> demos.haus </img>](https://demos.haus)

mattea-turic commented:
Thanks for your work on this @jw-can !

Answering your questions first:
> Could it make sense to move to the logo section pattern? [Logo section | Patterns | Vanilla documentation](https://vanillaframework.io/docs/patterns/logo-section) 

Yes :) I didn't want to add extra work for you to use the new pattern, but if you have the time that'd be great!

> The logo for Onetrail is not centered vertically. Could you please update the asset?

Hmm this is odd as the one I added in the AM [here](https://assets.ubuntu.com/manager/details?file_path=41834c82-onetrail_logo.png) is aligned with the rest. Also shares same height as the others, so not sure if this is maybe an issue that'll be resolved if we move to the logo section pattern. 

> The lines in “Managed packages” already didn’t line up before. Would you know if there’s a vanilla pattern for this section that should be used? Should I try to fix the line alignment?

The closest one would probably be [tiered list](https://vanillaframework.io/docs/patterns/tiered-list) in terms of layout, but it doesn't allow for the lists. 

Another option would be the [divided](https://vanillaframework.io/docs/patterns/divided-section) or [basic section](https://vanillaframework.io/docs/patterns/basic-section), and laying out the content like so:
<img width="400" height="800" alt="Screenshot 2025-11-13 at 6 09 25 PM" src="https://github.com/user-attachments/assets/e6856d4b-67fa-42c4-b4be-f629d8427fca" />

But I'll need to check with the stakeholder if they're okay with this design.

If not, then removing the lines is probably the simplest fix. 

---

Other notes:
**Hero:**
- [ ] It's lost its padding at the top, maybe this is a bug? I see the same is happening in the [original page](https://ubuntu.com/managed)
- [ ] Could you use a [muted rule](https://vanillaframework.io/docs/patterns/rule#muted) above the CTA too?

**What managed services customers say:**
- [ ] There's quite a bit of spacing here. Could I just verify you're using [this macro](https://vanillaframework.io/docs/patterns/quote-wrapper)?
- [ ] The logos are also quite small here – is there a preferred way I should export these so they're not tiny?

**Building blocks in our managed infra portfolio:**
- [ ] Could you use the [equal heights macro](https://vanillaframework.io/docs/patterns/equal-heights) here? Sorry if you already are :') 
- [ ] Pls add [image highlight](https://vanillaframework.io/docs/patterns/images#highlighted-image) too 
- [ ] Add a `<br>` following "our" (desktops only)
- [ ] On medium and small screens, the images aren't scaling down properly. Is this an assets issue?

**Our process:**
- [ ] I understand this was carried over from the old page, but could you apply [muted rules](https://vanillaframework.io/docs/patterns/rule#muted) rather than the default between the list items here?

**Value for money with managed infrastructure and applications**
- [ ] CTA is missing a chevron at the end ( Explore our managed application offering › )

**Add-on support for Managed Services:**
- [ ] Muted rule above the H5 headings pls

Sorry that some of these items are due to the old page feeling out of date, so I've just added in the design changes so it feels consistent throughout – or at least, more consistent, than it would feel otherwise

Thanks !!

---
@eliman11 
> ...not sure if you have a preference @mattea-turic?

Are we able to configure the table so it scrolls horizontally within its own viewport instead of causing overflow on the entire page?

eliman11 commented:
Hi @jw-can sorry one more comment - I noticed you changed the URLs for the /managed/apps and /managed/firefighting-support pages to /managed-infrastructure/. Could you please change it back to the current URLs? The stakeholders will make more changes to this bubble in the future, so we were asked to keep them as is for now. Thanks!

jw-can commented:
@eliman11 
> * [ ]  Could we have "What managed services customers say" heading span full width so it doesn't wrap
> *
Unfortunately this isn't supported by the pattern. Shall I discuss a potential change with the team and plan a follow-up PR if a change is agreed upon?

> I noticed you changed the URLs for the /managed/apps and /managed/firefighting-support pages to /managed-infrastructure/. Could you please change it back to the current URLs? The stakeholders will make more changes to this bubble in the future, so we were asked to keep them as is for now. Thanks!

Currently multiple pages live below the same folder:
<img width="413" height="332" alt="image" src="https://github.com/user-attachments/assets/d5937a08-1286-42fe-b53d-36078fc9136c" />
- https://ubuntu.com/managed
- https://ubuntu.com/managed/apps
- https://ubuntu.com/managed/firefighting-support
- https://ubuntu.com/managed/apps/kafka
- https://ubuntu.com/managed/contact-us
- https://ubuntu.com/managed/thank-you
Code-wise they are interdependent. It is possible to separate them into different folders, respecting your request for the URLs. However, this may become a bit brittle and I'd have to check with the sites tribe if this is something we want to do.
Let me know if I should verify or if maybe the stakeholders don't feel too strongly about the URL change for the bubble.

jw-can commented:
@eliman11 I've made the necessary adjustments. Could you please have a check?
@petesfrench since you already helped out here, could I bother you for a code & QA review? :pray:

eliman11 commented:
LGTM thanks for your patience on this @jw-can!

petesfrench commented on templates/managed-infrastructure/index.html:
```diff
@@ -209,13 +127,63 @@ <h2>Key benefits</h2>
+) -%}
+
+{%- if slot == 'signpost_image' -%}
+<img src="https://assets.ubuntu.com/v1/4f1e0548-esa_logo.png" alt="ESA logo">
```
```suggestion
<img src="https://assets.ubuntu.com/v1/4f1e0548-esa_logo.png" alt="ESA">
```

petesfrench commented on templates/managed-infrastructure/index.html:
```diff
@@ -209,13 +127,63 @@ <h2>Key benefits</h2>
+) -%}
+
+{%- if slot == 'signpost_image' -%}
+<img src="https://assets.ubuntu.com/v1/24693a21-tnm_logo.png" alt="TNM logo">
```
```suggestion
<img src="https://assets.ubuntu.com/v1/24693a21-tnm_logo.png" alt="TNM">
```

petesfrench commented on templates/managed-infrastructure/index.html:
```diff
@@ -308,61 +275,98 @@ <h3 class="p-heading--4">Telco cloud</h3>
+  items=[
+    {
+      "title_text": "Scalable software-defined storage",
+      "image_html":  "<img src='https://assets.ubuntu.com/v1/ad6ec304-scalable.png' class='p-image-container__image' width='284' height='426' alt='Storage illustration' />",
```
```suggestion
      "image_html":  "<img src='https://assets.ubuntu.com/v1/ad6ec304-scalable.png' class='p-image-container__image' width='284' height='426' />",
```

petesfrench commented on templates/managed-infrastructure/index.html:
```diff
@@ -308,61 +275,98 @@ <h3 class="p-heading--4">Telco cloud</h3>
+    },
+    {
+      "title_text": "Virtualization and private cloud management",
+      "image_html":  "<img src='https://assets.ubuntu.com/v1/9f7d88c5-virtualization.png' class='p-image-container__image' width='284' height='426' alt='Virtualization and cloud illustration' />",
```
```suggestion
      "image_html":  "<img src='https://assets.ubuntu.com/v1/9f7d88c5-virtualization.png' class='p-image-container__image' width='284' height='426' />",
```

petesfrench commented on redirects.yaml:
```diff
@@ -769,8 +769,8 @@ licensing/?: "https://canonical.com/legal/intellectual-property-policy"
 maas/?: "https://maas.io/"
+managed/?: "/managed-infrastructure"
 managed/kafka/?: "/managed/apps/kafka"
-managed/cassandra?: "/managed/apps/cassandra"
```
Should probably keep this redirect an point to /managed-infrastructure

petesfrench commented on templates/managed-infrastructure/index.html:
```diff
@@ -308,61 +275,98 @@ <h3 class="p-heading--4">Telco cloud</h3>
   <div class="row--50-50">
     <hr class="p-rule" />
     <div class="col">
       <h2>Our process</h2>
```
Not sure if it matters, but the number are right aligned with the `hr` above in the [design](https://www.figma.com/design/kEKHjfwg8ETTSVCNHRoAf7/ubuntu.com-managed-infrastructure---Sites?node-id=1-1474&m=dev)

jw-can replied:
Great catch!
I had a quick try but couldn't make this properly work without [stepped list](https://vanillaframework.io/docs/patterns/lists). @mattea-turic any thoughts about this? Shall we leave as is now?

petesfrench commented on templates/managed-infrastructure/form-data.json:
```diff
@@ -0,0 +1,81 @@
+        "title": "You innovate, we operate!",
+        "introText": "",
+        "formId": 4053,
+        "returnUrl": "/managed-infrastructure/thank-you#contact-form-success",
```
```suggestion
        "returnUrl": "/managed-infrastructure#contact-form-success",
```

jw-can commented:
> @jw-can We have been taking a phased approach will migrating all the forms, as there are so many. It also would've been marginally quicker for you. But what have done is also fine, it is not a priority!

I'm happy to make the adjustments, but I believe product will need to tell us first if they're fine with a default form?

jw-can commented:
> QA is good, just a couple notes in the code.
> 
> Can you also migrate the grid to use the [latest Vanilla grid pattern](https://vanillaframework.io/docs/patterns/grid#common-patterns-new) please

Thanks a lot for the review @petesfrench ! I've addressed your comments (except for the contact form and the "Our process" alignment, which I'm waiting for an answer from design). Could you please have another look? :pray:
